ga4gh/cli.py

```diff
@@ -266,10 +269,10 @@ def server_main(parser=None):
             description="GA4GH reference server")
     addGlobalOptions(parser)
     args = parser.parse_args()
-    frontend.configure(args.config_file, args.config)
+    frontend.configure(args.config_file, args.config, args.oidc_config_file)
     frontend.app.run(
-        host="0.0.0.0", port=args.port,
-        use_reloader=not args.dont_use_reloader)
+        host="localhost", port=args.port,
```

Comment:
Setting to 'localhost' might be overly restrictive, since some developers (e.g., me) might like to work on a remote server. However, I agree this is the right default behaviour, as we really don't want people putting this up on the internet. Perhaps we could add a command line argument for the host, with some docs explaining that you need to set this to something else if you want to use the server on the network?